engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSubscriptionTypeProcessor.java

```diff
@@ -105,9 +123,13 @@ void postConstruct() {
         } else {
             Action action = event.getAction();
             final EmailTemplate template = emailTemplateFactory.get(action.getBundle(), action.getApplication());
-            final boolean shouldSaveAggregation = Arrays.stream(EmailSubscriptionType.values())
-                    .filter(emailSubscriptionType -> emailSubscriptionType != EmailSubscriptionType.INSTANT)
-                    .anyMatch(emailSubscriptionType -> template.isSupported(action.getEventType(), emailSubscriptionType));
+            boolean shouldSaveAggregation;
+            if (useTemplatesFromDb) {
+                shouldSaveAggregation = templateRepository.isEmailAggregationSupported(action.getBundle(), action.getApplication(), NON_INSTANT_SUBSCRIPTION_TYPES);
+            } else {
+                shouldSaveAggregation = NON_INSTANT_SUBSCRIPTION_TYPES.stream()
```

Comment:
Not part of this PR, but i'm questioning now this mechanism.
Now that we have the event log we should probably take it from there.

Reply:
I want to make sure I understood your comment. Do you mean that we should use the payloads from the `Event#payload` rather than duplicating them into `email_aggregation`? If so, I agree with you 😃 The whole duplication followed by a purge process doesn't seem useful anymore since the DB contains the timestamp of the latest aggregation.

Reply:
We would gain in flexibility if we used `Event#payload`: each account could have a different aggregation time frame (1 day, 1 week, 2 hours...) with minimal code/DB changes.

Reply:
> Do you mean that we should use the payloads from `Event#payload` rather than duplicating them into `email_aggregation`?

That's what I meant :D